components/Mypage/mypageTitle.tsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/prop-types */
+import { Text } from '@chakra-ui/react';
+import styled from '@emotion/styled';
+import { AiOutlineLeft } from 'react-icons/ai';
+
+export default function MypageTitle({ ...props }) {
+  return (
+    <TitleContainer>
+      <AiOutlineLeft fontSize={30} />
+      <Text
+        fontSize="2xl"
+        marginLeft="4.5rem"
+        fontWeight="700"
+        paddingBottom="1rem"
+      >
```

Comment:
```suggestion
        fontWeight="700"
        paddingBottom="1rem"
        alignItems="center"
      >
```

중앙정렬 하면 조금 더 예뻐지는거 같아요